pkg/build/fuchsia.go

```diff
@@ -28,7 +31,7 @@ func (fu fuchsia) build(targetArch, vmType, kernelDir, outputDir, compiler, user
 	}
 	for src, dst := range map[string]string{
 		"out/" + arch + "/obj/build/images/fvm.blk": "image",
-		".ssh/pkey":                                 "key",
+		".ssh/pkey":                                 "sshkey",
```

Comment:
I am confused. The build function is supposed to produce outputs that the caller will understand. If the caller expects "sshkey" then fuchsia build would have never been worked, but it worked. If the caller expects the key to be in the "key" file, than this will break build. What I am missing?

Reply:
At the time I sent this out, this *seemed* to resolve an SSH connection issue I was having when running "syz-manager" for Fuchsia locally.

In hindsight I think perhaps there were confounding factors; accidentally running with an old syz-manager version or something.

We still seem to be getting SSH failures for Fuchsia on the syz-ci bots, but this pull request isn't the solution to those.  Myself and mvanotti@ are planning to finish root-causing the SSH issue tomorrow so hopefully we'll have a real fix by then.